secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc

```diff
@@ -26,22 +26,24 @@ include::modules/op-creating-mounting-kms-authentication-token-secret.adoc[level
 include::modules/op-enabling-tekton-chains-to-operate-only-in-selected-namespaces.adoc[leveloffset=+2]
 
 include::modules/op-signing-secrets-in-tekton-chains.adoc[leveloffset=+1]
+include::modules/op-chains-generating-x509-secret.adoc[leveloffset=+2]
 include::modules/op-chains-signing-secrets-cosign.adoc[leveloffset=+2]
 include::modules/op-chains-signing-secrets-skopeo.adoc[leveloffset=+2]
 include::modules/op-chains-resolving-existing-secret.adoc[leveloffset=+2]
 
 include::modules/op-authenticating-to-an-oci-registry.adoc[leveloffset=+1]
 
 include::modules/op-creating-and-verifying-task-run-signatures-without-any-additional-authentication.adoc[leveloffset=+1]
-=== Additional resources
 
-* xref:signing-secrets-in-tekton-chains_{context}[]
-* xref:configuring-tekton-chains_{context}[]
+[role="_additional-resources"]
+.Additional resources
+* xref:../secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc#signing-secrets-in-tekton-chains_using-tekton-chains-for-openshift-pipelines-supply-chain-security[Secrets for signing data in {tekton-chains}]
+* xref:../secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc#configuring-tekton-chains_using-tekton-chains-for-openshift-pipelines-supply-chain-security[Configuring {tekton-chains}]
 
 include::modules/op-using-tekton-chains-to-sign-and-verify-image-and-provenance.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
-[id="additional-resources-tekton-chains"]
+[id="additional-resources_{context}"]
 == Additional resources
-
+* xref:../secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc#signing-secrets-in-tekton-chains_using-tekton-chains-for-openshift-pipelines-supply-chain-security[Secrets for signing data in {tekton-chains}]
 * xref:../install_config/installing-pipelines.adoc#installing-pipelines[Installing {pipelines-shortname}]
```

Comment:
Although out of context of your intended changes, probably good to update the second link as well.

Reply:
The second link seems correct to me here